rosdep/base.yaml

```diff
@@ -1388,6 +1388,81 @@ ifstat:
   fedora: [ifstat]
   gentoo: [net-analyzer/ifstat]
   ubuntu: [ifstat]
+ignition-citadel:
+  debian:
+    buster: [ignition-citadel]
+  ubuntu:
+    focal: [ignition-citadel]
+ignition-cmake2:
+  debian:
+    buster: [libignition-cmake2-dev]
```

Comment:
Rosdep rules try to match the debian package name, which I thought was in the contributing guide but now I can't find it. Along those lines, I think it would make sense to separate the `dev` and non-`dev` keys. So `libignition-cmake2` is one key and `libignition-cmake2-dev` is another.

Reply:
I've been following @j-rivero 's pattern started here: https://github.com/osrf/osrf-rosdep/blob/master/ignition/ignition.yaml

What do you think, Jose?

Reply:
> What do you think, Jose?

If the rest of the rosdep archive is using that model, I think we should follow it and change ours. I thought all rosdep keys mainly pointed to `-dev` packages but that could be just me.

Reply:
e9bc4cb49c29c97494b51e838aec1e6f9cc5d019